.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+    "rust-analyzer.cargo.loadOutDirsFromCheck": true,
```

Comment:
Based on https://code.visualstudio.com/docs/languages/json#_json-with-comments VSCode should support comments in these files. Can you add a comment explaining what this does and linking to https://github.com/rust-analyzer/rust-analyzer/issues/6448?